client/js/components/procedure/AdministrationImport/AdministrationImport.vue

```diff
@@ -160,12 +165,17 @@ export default {
               title: addon.options.title
             })
           }
-      })
-    },
+        })
+    }
   },
 
   mounted () {
-    this.loadComponents('import.tabs')
+    const loadComponents = this.loadComponents('import.tabs')
+
+    Promise.allSettled([loadComponents])
```

Comment:
Nitpick: In other places we place the functions inside the brackets, like so:

```suggestion
    Promise.allSettled([this.loadComponents('import.tabs')])
```
Personally i find this more concise, what do you think?